phpstan.neon

```diff
@@ -293,3 +293,8 @@ parameters:
             paths:
                  - packages/package-builder/src/Parameter/ParameterProvider.php # 26
                  - packages/symfony-static-dumper/src/HttpFoundation/ControllerContentResolver.php # 37
+
+        -
+            message: '#Use explicit method names over dynamic#'
+            paths:
+                 - packages/changelog-linker/tests/FileSystem/ChangelogFileSystem/ChangelogFileSystemTest.php # 43
```

Comment:
This should be resolved, using exact method name

Reply:
we use phpunit 8 and 9, which method `assertMatchesRegularExpression` used in PHPUnit 9 and `assertRegExp` in PHPUnit 8, which deprecated if used in PHPUnit 9.

Reply:
Indeed. The exact method can be used directly.

```php
if (method_exists(.....)) {
    $this->methodName()
}
```


It's less magic and IDE aware

Reply:
I updated with `assertStringContainsString` with read local file.